config/config.go

```diff
@@ -395,7 +398,8 @@ func decodeConfig(content []byte, file string) (*Config, error) {
 		return nil, fmt.Errorf("invalid format: %s", format)
 	}
 	if err != nil {
-		log.Printf("[ERR] (config) failed to decode %s", format)
+		logger.Error("failed to decode config",
+			"file_format", format, "error", err)
```

Comment:
Just wondering if you have a line length limit set in your IDE and if so how many characters do you have it at?